sc2reader/factories/plugins/replay.py

```diff
@@ -100,21 +100,30 @@ def APMTracker(replay):
     above actions divided by the number of seconds played by the player (not
     necessarily the whole game) multiplied by 60.
     """
+
+    speed_multiplier = 1
+    if replay.expansion == 'LotV':
+        speed_multiplier = 1.4
+
+    game_seconds_per_second = 1.4
+    if replay.expansion == 'LotV':
+        game_seconds_per_second = 1.4
```

Comment:
Oh, that's unintentional - should have been 1 for LotV - will update it right away.